lib/request-wrapper.ts

```diff
@@ -346,6 +368,40 @@ export class RequestWrapper {
     return this.axiosInstance;
   }
 
+  private static getRetryConfig(
+    axiosInstance: AxiosInstance,
+    retryConfig?: RetryOptions
+  ): rax.RetryConfig {
+    const config: rax.RetryConfig = {
+      instance: axiosInstance,
+      backoffType: 'exponential',
+      checkRetryAfter: true,
+    };
+    if (retryConfig) {
+      if (typeof retryConfig.maxRetries === 'number') {
+        config.retry = retryConfig.maxRetries;
+      }
+      if (typeof retryConfig.maxRetryInterval === 'number') {
+        config.maxRetryDelay = retryConfig.maxRetryInterval;
+      }
+    }
+    return config;
+  }
+
+  public enableRetries(retryConfig?: RetryOptions) {
+    this.axiosInstance.defaults.raxConfig = {
+      instance: this.axiosInstance,
+    };
+    this.retryConfig = RequestWrapper.getRetryConfig(this.axiosInstance, retryConfig);
```

Comment:
suggested name changes:
```suggestion
    this.raxConfig = RequestWrapper.getRaxConfig(this.axiosInstance, retryOptions);
```

Reply:
Done!